File: Results/DER/Translation/translation_output/w_test/magicoder/codenet/python/java/p02552.java
<｜begin▁of▁sentence｜>You are an exceptionally intelligent coding assistant that consistently delivers accurate and reliable responses to user instructions.

@@ Instruction
Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
1

Expected output:
0


```
print(1 - int(input()))
```

@@ Response
Here is the Java version of the given Python code:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int input = scanner.nextInt();
        System.out.println(1 - input);
    }
}
```

You can test this code with the following test input:

```
1
```

And the expected output will be:

```
0
```<｜end▁of▁sentence｜>

File: Results/DER/Translation/translation_output/w_test/magicoder/codenet/java/python/p02552.py
<｜begin▁of▁sentence｜>You are an exceptionally intelligent coding assistant that consistently delivers accurate and reliable responses to user instructions.

@@ Instruction
Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
1

Expected output:
0


```
import java.util.Scanner;

/**
 * @author yadavd5
 *
 */
public class p02552 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		if(n==0)
			System.out.println("1");
		else
			System.out.println("0");
	}
}
```

@@ Response
```python
# Python code

# Get input from user
n = int(input())

# Check if n is 0
if n == 0:
    print("1")
else:
    print("0")
```

Test input:
1

Expected output:
0<｜end▁of▁sentence｜>
